docs/README.md

```diff
@@ -2,19 +2,19 @@
 
 ## Overview
 
-The Cloud Operations Sandbox is intended to make it easy for you to deploy and run a non-trivial application that lets you explore the Google Cloud Platform services, particularly the [Ops Management](http://cloud.google.com/products/operations) (formerly Stackdriver) product suite. Ops Management is a suite of tools that helps you gain full observability into your code and applications.
+The Cloud Operations Sandbox is intended to make it easy for you to deploy and run a non-trivial application that lets you explore the Google Cloud Platform services, particularly the [Cloud Operations](http://cloud.google.com/products/operations) (formerly Stackdriver) product suite. Cloud Operationsis a suite of tools that helps you gain full observability into your code and applications.
 
 The Hipster Shop application used in the sandbox is intended to be sufficiently complex such that you can meaningfully experiment with it, and the Sandbox automatically provisions a new demo cluster, configures and deploys Hipster Shop, and simulates real users.
 
-With the Sandbox running, you can experiment with various Ops Management tools to solve problems and accomplish standard SRE tasks in a sandboxed environment without impacting your production monitoring setup.
+With the Sandbox running, you can experiment with various Cloud Operationstools to solve problems and accomplish standard SRE tasks in a sandboxed environment without impacting your production monitoring setup.
```

Comment:
`Cloud Operationstools` -> `Cloud Operations tools` -> 

Is your IDE getting formatted weirdly after you push to Github? This happened to me at one point...

Reply:
How did you solve it?

Reply:
I think I was using a different IDE. Now I use VScode with default settings.